kratos/tests/test_embedded_skin_mapping.py

```diff
@@ -0,0 +1,85 @@
+﻿from __future__ import print_function, absolute_import, division
+
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics
+import os
+
+def GetFilePath(fileName):
+    return os.path.join(os.path.dirname(os.path.realpath(__file__)), fileName)
+
+class TestEmbeddedSkinMapping(KratosUnittest.TestCase):
+
+    def test_embedded_skin_mapping(self):
+        model = KratosMultiphysics.Model()
+
+        # Set the problem domain using the structured mesh generator process
+        model_part = model.CreateModelPart("ModelPart")
+        model_part.AddNodalSolutionStepVariable(KratosMultiphysics.PRESSURE)
+        model_part.AddNodalSolutionStepVariable(KratosMultiphysics.DISTANCE)
+
+        problem_domain = KratosMultiphysics.Quadrilateral2D4(
+            KratosMultiphysics.Node(1, -0.25, -0.75, 0.0),

```

Comment:
this is not correct...where do these nodes belong? (answer: to no modelpart!)

Reply:
you should create the nodes using the function CreateNewNode of the target modelpart

Reply:
These nodes are for the creation of the geometry provided to the mesh generator. Actually you don't want them to belong to the model part since the mesh generator would be the one creating that corner nodes.

Reply:
ok